app/editor/index.tsx

```diff
@@ -202,6 +206,8 @@ export class Editor extends React.PureComponent<
     this.events.on(EventType.blockMenuClose, this.handleCloseBlockMenu);
     this.events.on(EventType.emojiMenuOpen, this.handleOpenEmojiMenu);
     this.events.on(EventType.emojiMenuClose, this.handleCloseEmojiMenu);
+    this.events.on(EventType.mentionMenuOpen, this.handleOpenMentionMenu);
+    this.events.on(EventType.mentionMenuClose, this.handleCloseMentionMenu);
```

Comment:
Need to add it to the list to refactor this mess to something less repetitive, probably an `openMenu` enum at a minimum as it's not possible to have multiple open at once.